lms/static/karma_lms.conf.js

```diff
@@ -28,6 +28,7 @@ var options = {
     sourceFiles: [
         {pattern: 'coffee/src/**/!(*spec).js'},
         {pattern: 'course_bookmarks/**/!(*spec).js'},
+        {pattern: 'course_search/**/!(*spec).js'},
```

Comment:
Good question, but no. The course experience JavaScript is bundled by Webpack automatically.